returnn/datasets/postprocessing.py

```diff
@@ -186,6 +186,13 @@ def init_seq_order(
                 pass  # some datasets don't know their num_seqs
         return True
 
+    def get_current_seq_order(self):
+        """:return: current seq order of wrapped dataset, if map_seq_stream is not used"""
+        if self._map_seq_stream is not None:
+            raise util.OptionalNotImplementedError
```

Comment:
That's wrong. That's not something which is not implemented (also optionally not implemented does anyway not make sense). That's something which is just not allowed / not possible.
```suggestion
            raise Exception(f"{self} get_current_seq_order not allowed with map_seq_stream")
```